app-web/src/components/Form/ExistingResourceForm.js

```diff
@@ -1,93 +1,199 @@
 import React, { useMemo, useState, Fragment } from 'react';
-import { SelectDropdown } from './form';
-import { graphql, useStaticQuery } from 'gatsby';
+import { graphql, useStaticQuery, StaticQuery } from 'gatsby';
 import { flattenGatsbyGraphQL } from '../../utils/dataHelpers';
+import {
+  TextInput,
+  SelectDropdown,
+  StylesWrapper,
+  StyledSuccessMessage,
+  StyledErrorMessage,
+} from './form';
+import StyledButton from '../UI/Button/Button';
+import { Form } from 'react-final-form';
+import arrayMutators from 'final-form-arrays';
+import { FieldArray } from 'react-final-form-arrays';
 
 export const ExistingResourceForm = () => {
-  const [currentFormValue, setFormValue] = useState('');
-  let content;
+  const topicName = location.pathname.replace('/editTopic/', '').replace(/-/g, ' ');
```

Comment:
theres better ways of doing this. Paths are also carried in the `location` prop when using client side only routes. 

So you could do something like `<EditTopic path="/editTopic/:name" />`

and reference it within the `location` prop